types/react-native-modals/index.d.ts

```diff
@@ -32,7 +33,7 @@ export interface ModalContentProps {
 export interface BackdropProps {
     visible: boolean;
     opacity: number;
-    onPress?: (() => void) | undefined;
+    onPress?: () => void | undefined;
```

Comment:
This isn't correct; the parens are required otherwise this is a function that returns `void | undefined`. The same applies to all of the other cases below